newrelic.js

```diff
@@ -44,6 +44,18 @@ exports.config = {
    * attributes include/exclude lists.
    */
   allow_all_headers: true,
+  application_logging: {
+    /**
+     * When true, logs generated by your application will be forwarded to New Relic One.
```

Comment:
this is not techically true.  you need to have both `application_logging.enabled` and `application_logging.forwarding.enabled` set to true.  The ticket #1150 describes what to add to this default, this is missing the forwarding bit and included the application_logging.enabled flag even though that defaults to true in agent

Reply:
Updated!